### PR TITLE
Add CRA capabilities

### PR DESCRIPTION
**Pull request by jose-hernandez2**
New tasks for CRA capabilities added. I'm not sure about the changes in the bindings, templates and values.yaml, so please check that more in detail.

### CODE REVIEW

File: tekton/helm/templates/cicd/base/05-pipelines/ci-pipeline-general.yaml [hunk 1]
```diff
@@ -119,6 +134,30 @@ spec:
     - name: maven-settings
       workspace: maven-settings
   # ##################
+  # CRA scan
+  - name: cra-scan
+    params:
+    - name: CONTEXT_DIR
+      value: '$(params.RUN_ID)'
+    - name: CHECK_ENABLED
+      value: "true"
```
Comment (haf-tech) [inline tekton/helm/templates/cicd/base/05-pipelines/ci-pipeline-general.yaml]: please extract this to a param and use this here too (params.CRA_ENABLED)

**Review by haf-tech — CHANGES_REQUESTED** [review]
small opti needed

File: tekton/helm/templates/cicd/base/05-pipelines/ci-pipeline-general.yaml (hunk 1 above)
Comment (jose-hernandez2) [reply tekton/helm/templates/cicd/base/05-pipelines/ci-pipeline-general.yaml]: Please check commit 8e95a67a6391e26908a08e84e493f776a276c3d4

**Review by haf-tech — APPROVED** [review]
lgtm

**Pull request merged**